src/utils/api.ts

```diff
@@ -0,0 +1,21 @@
+interface body {
```

Comment:
body should be defined by some HTML/JS generated thing or not at all, this is a general API wrapper so we cant conform to just the name and email